README.rst

```diff
@@ -26,6 +26,18 @@ It contains the following items:
 -  `Third-Party code <third-party-code.rst>`_: The circumstances under which the
    PHP project will use PHP code written by third parties in the ecosystem.
 
+***********************
+ Requirements Notation
+***********************
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in `BCP 14
```

Comment:
In this document, or in these documents? Not a real problem, but we ought to fix that.

Reply:
Changed